pkg/transparentproxy/iptables/builder/builder.go

```diff
@@ -240,13 +216,13 @@ func RestoreIPTables(cfg config.Config) (string, error) {
 		}
 	}
 
-	output, err := restoreIPTables(cfg, dnsIpv4, false)
+	output, err := restoreIPTables(context.TODO(), cfg, dnsIpv4, false)
```

Comment:
Can we wire the ctx through shouldn't be that complicated...

Reply:
done